libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -59,7 +59,7 @@ const AP_Param::GroupInfo AP_OAPathPlanner::var_info[] = {
     // @DisplayName: Options while recovering from Object Avoidance
     // @Description: Bitmask which will govern vehicles behaviour while recovering from Obstacle Avoidance (i.e Avoidance is turned off after the path ahead is clear).   
     // @Bitmask{Rover}: 0: Reset the origin of the waypoint to the present location, 1: log Dijkstra points
-    // @Bitmask{Copter}: 1: log Dijkstra points
+    // @Bitmask{Copter}: 1:log Dijkstra points, 2:Allow fast waypoints (Dijkastras only)
```

Comment:
Yes, I agree this new behaviour is better from a user point of view so we should make it the default.  So that means either reversing the meaning of the bit (e.g. disable instead of enable) or setting the option param default to 2.  Let's do this in a follow-up commit after some flight testing.  As long as we make the change before we release 4.5 then it should be OK.